kuma/javascript/src/document.jsx

```diff
@@ -237,6 +239,7 @@ function DocumentPage({ document }: DocumentProps) {
     return (
         <>
             <Header document={document} />
+            <Banners banners={banners} />
             <TaskCompletionSurvey document={document} />
```

Comment:
Looking at this again now, I reckon it would be better to have these two at the end of the list of components. From an accessibility perspective, it would be better to have the `Titlebar` and `Breadcrumbs` in the document structure and tab order before the two banners.